### PR TITLE
Recordings dir and env vars to disable plugins

### PR DESCRIPTION
**Pull request by bblaszkow06**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,6 +92,13 @@ to the original configuration files in order to provide more consistent usage.
 - ICE_ENFORCE_LIST - choose which interfaces should be explicitly used by the gateway for the purpose of ICE candidates gathering (e.g. "eth0,192.168.0.1"), empty by default
 - ICE_IGNORE_LIST - choose which interfaces should be ignored by the gateway for the purpose of ICE candidates gathering (e.g. "eth0,192.168.0.1"), (default=vmnet)
 
+### Loaded libraries
+
+- DISABLED_PLUGINS - disables plugins, e.g. "libjanus_voicemail.so,libjanus_recordplay.so"
+- DISABLED_TRANSPORTS - disables transport libraries (default="libjanus_pfunix.so")
```
Comment (mspanc) [inline README.md]: Please make all of them empty, it was my addition to disable pfunix by default. In original janus config each default is empty and we should stick to this.

File: templates/janus.jcfg [hunk 2]
```diff
@@ -199,8 +202,11 @@ media: {
 # be used for gathering candidates, and enable or disable the
 # internal libnice debugging, if needed.
 nat: {
+	{{ if .Env.STUN_SERVER }}
```
Comment (mspanc) [inline templates/janus.jcfg:205]: I'd suggest checking for both STUN_SERVER and STUN_PORT

File: templates/janus.jcfg [hunk 3]
```diff
@@ -295,7 +303,7 @@ plugins: {
 # to identify the transports to disable. By default all available
 # transports are enabled and loaded at startup.
 transports: {
-	disable = "libjanus_pfunix.so"
+	disable = "{{ default .Env.DISABLED_TRANSPORTS "libjanus_pfunix.so" }}"
```
Comment (mspanc) [inline templates/janus.jcfg]: make default empty

**Review by mspanc — CHANGES_REQUESTED** [review]

File: templates/janus.jcfg (hunk 2 above)
Comment (bblaszkow06) [reply templates/janus.jcfg:205]: Well, I don't think its necessary - If STUN_SERVER is not set STUN_PORT does nothing and #3 can be fixed by adding a pair of quotes

**Pull request merged**